Validation/RecoTrack/python/TrackValidation_cff.py

```diff
@@ -282,24 +282,51 @@ def _addSelectorsByOriginalAlgoMask(modules, midfix, algoParam):
     trackValidatorAllTPEfficStandalone.label.remove(_label)
 
 # sequences
-tracksPreValidationStandalone = cms.Sequence(
-    tracksPreValidation +
+tracksValidationSelectorsStandalone = cms.Sequence(
     tracksValidationSelectorsByOriginalAlgoStandalone +
     tracksValidationSelectorsByAlgoMaskStandalone +
     tracksValidationSelectorsFromPVStandalone
 )
-tracksValidationStandalone = cms.Sequence(
+tracksPreValidationStandalone = cms.Sequence(
     ak4PFL1FastL2L3CorrectorChain+
-    tracksPreValidationStandalone+
+    tracksPreValidation +
+    tracksValidationSelectorsStandalone
+)
+trackValidatorsStandalone = cms.Sequence(
     trackValidatorStandalone +
     trackValidatorFromPVStandalone +
     trackValidatorFromPVAllTPStandalone +
     trackValidatorAllTPEfficStandalone
 )
+tracksValidationStandalone = cms.Sequence(
+    tracksPreValidationStandalone+
+    trackValidatorsStandalone
+)
+
+### TrackingOnly mode (i.e. MTV with DIGI input + tracking-only reconstruction
+
+# selectors
+tracksValidationSelectorsTrackingOnly = tracksValidationSelectors.copyAndExclude([ak4JetTracksAssociatorExplicitAll,cutsRecoTracksAK4PFJets]) # selectors using track information only (i.e. no PF)
+
+# MTV instances
+trackValidatorTrackingOnly = trackValidatorStandalone.clone()
+trackValidatorTrackingOnly.label.remove("cutsRecoTracksAK4PFJets")
+
+# sequences
+tracksPreValidationTrackingOnly = tracksPreValidation.copy()
+tracksPreValidationTrackingOnly.replace(tracksValidationSelectors, tracksValidationSelectorsTrackingOnly)
+tracksPreValidationTrackingOnly += tracksValidationSelectorsStandalone
+
+trackValidatorsTrackingOnly = trackValidatorsStandalone.copy()
+trackValidatorsTrackingOnly.replace(trackValidatorStandalone, trackValidatorTrackingOnly)
+
+tracksValidationTrackingOnly = cms.Sequence(
+    trackValidatorsTrackingOnly
+)
 
 
 ### 'slim' sequences that only depend on track and tracking particle collections
-tracksValidationSelectorsSlim = tracksValidationSelectors.copyAndExclude([cutsRecoTracksBtvLike,ak4JetTracksAssociatorExplicitAll,cutsRecoTracksAK4PFJets])
+tracksValidationSelectorsSlim = tracksValidationSelectorsTrackingOnly.copyAndExclude([cutsRecoTracksBtvLike])
```

Comment:
@lveldere After this, the only difference between "trackingOnly" and "slim" would be that the former includes "BTV-like" track selection while the latter does not. I don't mind keeping "slim", but thought to ask anyway if keeping it is motivated?